src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -1787,26 +1801,20 @@ def _interp_az_za_map_coordinates(
         if spline_opts is None or not isinstance(spline_opts, dict):
             spline_opts = {}
 
-        _az_array = (
-            (az_array - phi_use.min())
-            / (phi_use.max() - phi_use.min())
-            * (phi_use.size - 1)
-        )
-        _za_array = (
-            (za_array - theta_use.min())
-            / (theta_use.max() - theta_use.min())
-            * (theta_use.size - 1)
-        )
+        az_array -= phi_use.min()
+        az_array /= (phi_use.max() - phi_use.min()) / (phi_use.size - 1)
+
+        za_array -= theta_use.min()
+        za_array /= (theta_use.max() - theta_use.min()) / (theta_use.size - 1)
```

Comment:
Minor nitpick: I'd find these divide lines easier to read if they were multiplied by the reciprocal. I have to think harder about what the two divides mean.

Reply:
Np. Updated.